commit 5a55f882cab2fe3387edbbce7f31f10cd56714b2
Author: Reto Gantenbein <reto.gantenbein@linuxmonk.ch>
Date:   Fri Oct 27 07:31:49 2017 +0200

    Revert "Temporary workaround for log15 API breakage"
    
    This reverts commit 30c859499e24ba6255508cd664c09cf4c1731bbe.

diff --git a/shared/logging/log_posix.go b/shared/logging/log_posix.go
index 5ada808..ffc55da 100644
--- a/shared/logging/log_posix.go
+++ b/shared/logging/log_posix.go
@@ -3,8 +3,6 @@
 package logging
 
 import (
-	slog "log/syslog"
-
 	log "gopkg.in/inconshreveable/log15.v2"
 )
 
@@ -15,11 +13,11 @@ func getSystemHandler(syslog string, debug bool, format log.Format) log.Handler
 		if !debug {
 			return log.LvlFilterHandler(
 				log.LvlInfo,
-				log.Must.SyslogHandler(slog.LOG_INFO, syslog, format),
+				log.Must.SyslogHandler(syslog, format),
 			)
 		}
 
-		return log.Must.SyslogHandler(slog.LOG_INFO, syslog, format)
+		return log.Must.SyslogHandler(syslog, format)
 	}
 
 	return nil
